### DISCUSSION TITLE
Install bracket on Synology NAS or RPi4

### DISCUSSION THREAD

**Discussion by aledre** [evroon/bracket#935] [Q&A]
Hi everyone,
I'm new to programming, very new :) But I had the opportunity to deal with the translation of programs, to bring the Italian language into many projects.
Evroon has activated the possibility of having this project translated for me on crowdin, but to do a correct translation, I need to run the program. I can't start the program on a Raspberry, I can clone the repository but when I launch "docker compose up -d" it returns a series of errors. The raspberry has ubuntu installed, and I have docker, docker-compose and git clone installed. I can also try using my synology nas, I have other dockers installed, and I (in ssh) pulled the backend and fronend, but I don't know how to configure the dockers. Can anyone help me?

**Comment by aledre** [comment]
in RPI whem I launch sudo docker-compose up -d it return "no matching manifest for linux/arm64/v8 in the manifest list entries" so I try to install qemu and add platform: linux/amd64 in yml, but it don't work with command sudo docker-compose up -d --platform linux/amd64 command

**Comment by aledre** [comment]
in synology nas I don't know witch variables I need to add, now I have
![image](https://github.com/user-attachments/assets/da2cd88a-c1da-4f5f-9e12-0f826f405ec5)

**Comment by evroon** [comment]
Yes the docker image doesn't support ARM yet, someone tried to add support for it in the past, but it doesn't work yet, so it's still a work in progress

**Reply by aledre** [reply]
thanks for reply, I'll try on synology, if someone point me to the right direction with variables needed
Ah, I add italian language to crowdin, take a look... when I can try bracket I'll correct eventual mistakes

**Comment by aledre** [comment]
Ok I've create a VM debian 12 with virtualbox and all work, sorry for some stupid questions

**Reply by evroon** [reply]
Haha they weren't stupid questions, good to hear it works now

**Comment by evroon** [comment]
Bracket can now be installed on RPis as well with Docker: https://github.com/evroon/bracket/pull/1120
Release v2.2.0 contains such a multi-architecture docker image.
